src/commons/workload_mapping.py

```diff
@@ -39,4 +40,5 @@
     'object_range_read': [object_operations.TestS3Object, 'execute_object_workload'],
     'copy_object_fix_size': [copy_object.TestS3CopyObjects, 'execute_copy_object_workload'],
     'mix_object_ops': [mix_object_operations.TestMixObjectOps, 'execute_mix_object_workload']
```

Comment:
invalid syntax (F999). comma missing in 42.

Reply:
corrected now.